src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -41,7 +36,7 @@ const styles = StyleSheet.create({
     borderRadius: 16
   },
   anonImageBg: {
-    backgroundColor: colors.anon_primary
+    backgroundColor: COLORS.holyTosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color has been updated to use the new `COLORS` import. This is a good change as it maintains consistency across the codebase.
```diff
-    backgroundColor: colors.anon_primary
+    backgroundColor: COLORS.holyTosca
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->